receiver/kafkareceiver/testdata/config.yaml

```diff
@@ -60,4 +60,41 @@ kafka/rebalance_strategy:
     topic: otlp_logs
     encoding: otlp_proto
   group_rebalance_strategy: sticky
-  group_instance_id: test-instance
\ No newline at end of file
+  group_instance_id: test-instance
+kafka/message_marking:
+  metrics:
+    topic: otlp_metrics
+    encoding: otlp_proto
+  traces:
+    topic: otlp_spans
+    encoding: otlp_proto
+  logs:
+    topic: otlp_logs
+    encoding: otlp_proto
+  message_marking:
+    after: true
+    on_error: true
+    on_permanent_error: true
```

Comment:
Should we make this one `on_permanent_error: false`, to show that if the field is explicitly configured it will not inherit the value of `on_error`?

Reply:
I agree with what you said. I made the correction in the commit below.
- commit : https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/41883/commits/67b9bad2a6f0685db3fcb27be6aa909731f7fc09